packages/xrpl/test/integration/index.ts

```diff
@@ -1,62 +1 @@
-/* eslint-disable import/export -- Tells webpack which files exist. */
-
-// These go first because they're affected by the `ledger_accept`s
-export * from './transactions/escrowFinish.test'
-export * from './transactions/escrowCancel.test'
```

Comment:
Does this new setup mess up the escrow fix here? (We originally needed them in the beginning to avoid ledger accept's from other tests creating a race condition where the escrow was unable to be finished I believe)

Reply:
I think my last fix avoids the problem

Reply:
Agreed.  Also this previous fix in the index.js file was only being used by the browser tests.